wp5_rag_explainer.md

```diff
@@ -0,0 +1,79 @@
+> - You can use "closed-source" LLMs hosted by large organisations - this is easy, and reasonably cheap, but you have less control and isn't suitable for sensitive applications
+> - You can host your own open-source models - this requires more infrastructure and overall cost - however you have complete control and can further train the model, run it your our own hardware etc.
+
+
+In the last year Large Language Models have exploded in popularity - these are **Artificial Intelligence tools**, which can seemingly answer any question you submit to them - and even better, **you can converse with them using normal everyday language**. 
+
+LLMs are trained on large bodies of data, learning the relationships between words and those that come before and after them - the end result is a model which when you submit a text string to it, it will predict the "tokens" (basically characters) that come next, one at a time, until the end result is whole sentences and paragraphs. It's important to remember that usually, under-the-hood, all it is doing is predicting what character comes next.
+
+### Closed-source vs Open-source models
+
+The state of the art models are huge, and require vast amounts of compute, manpower (and hence time and money) to train, with the latest models having 100s of billions (or even trillions) of parameters, having consumed potentially millions of documents with costs in the regions of 100's millions of dollars. These models are also often closed source, meaning that the company has not revealed exactly how the model was trained, or released the files for the model itself.
+
+However, it's possible to get decent performance with far less: many "open source" models, such as those from Mistral performs well with 70B (70 Billion) parameters and took *only* $500,000 to train. The cost to train models with acceptable levels of performance will only decrease as techniques improve and become more efficient.
+
+70B parameters is still very large, requiring special hardware to run - however, improvements in techniques are leading to even smaller models ~7B which can run on *normal* laptops, though the quality of the outputs of these models will probably be worse than the larger models. However, a huge benefit of open source is that you can adapt the model to your needs e.g. you can do further training (fine-tuning).
+
+It's important to to note that the closed-source models often run on servers held elsewhere - you submit your question to them, and it returns the answers. This means that's not suitable for many more sensitive use-cases (at least without a close partnership with the organisation running the model). Open source models by their nature can be run on hardware we own, and so can be used for any use-case.
+
+### Cost to run
+
+State of the art (closed-source models) cost something in the region of ~$10 per million "tokens" (similar to words) - so ~ 240,000 words/£.
```

Comment:
Want to be consistent with your definition of tokens. Previously you said "basically characters". Using "Similar to words" instead here adds confusion.

Reply:
Good point! They're words, not characters - I'll correct that!